src/components/consultation-pad-contents/consultation-pad-contents.tsx

```diff
@@ -60,14 +72,21 @@ export function ConsultationPadContents({closeConsultationPad}) {
         onChange={event => setConsultationText(event.target.value)}
         labelText=""
         ref={input => input && input.focus()}
-        value={consultationText}
+        value={
+          recordedText != ''
+            ? `${consultationText} ${recordedText}`
+            : consultationText
```

Comment:
recordedText? appendRecordedText() : consultationText
appendRecordedText() {
   //put append logic
}

Reply:
Made the changes here.